dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSTLTests.java

```diff
@@ -166,7 +171,15 @@ public void testFormatTags() throws Exception {
         WebResponse response = wc.getResponse(request);
         LOG.info("Servlet response : " + response.getText());
 
-        assertTrue("Something went wrong with format tags", response.getText().contains("Feb 17, 2021"));
+        Date today = new Date();
+        DateFormat dateFormatter = DateFormat.getDateInstance(DateFormat.MEDIUM, Locale.US);
+        String dateOut = dateFormatter.format(today);
+
+        assertTrue("Something went wrong with format tags. Did not find expected string: " + dateOut,
+            response.getText().contains(dateOut));
+
+        assertTrue("Something went wrong with the parse tags!",
+            response.getText().contains("Sat Jan 01 00:00:00")); 
```

Comment:
Just looking for date excluding year here to avoid timezone intermittent failures.

Reply:
EST 2000 vs PST 2000 etc